edx_proctoring/tests/test_student_view.py

```diff
@@ -717,40 +716,26 @@ def test_get_studentview_submitted_status(self):
                 'default_time_limit_mins': 90
             }
         )
-        self.assertIn(self.proctored_exam_waiting_for_app_shutdown_msg, rendered_response)
+        self.assertIn(self.proctored_exam_submitted_msg, rendered_response)
 
-        reset_time = datetime.now(pytz.UTC) + timedelta(minutes=2)
```

Comment:
Search for `last_poll_timestamp`, like above, and delete code that is setting (and probably using) it.  Thanks!